droidlet/dashboard/web/src/components/Timeline/TimelineUtils.js

```diff
@@ -48,3 +50,38 @@ export function handleClick(stateManager, item) {
     .getItemsById("timelineDetails")[0]
     .addChild(config);
 }
+
+export function handleSearch(stateManager, pattern) {
```

Comment:
why put this into util file? Seems that it either belongs to timeline module or a timeline_search module if you really need to separate it out.